hack/verify-generated-files-remake.sh

```diff
@@ -159,8 +167,8 @@ assert_clean
 touch staging/src/k8s.io/code-generator/cmd/defaulter-gen/main.go
 touch "${STAMP}"
 make generated_files >/dev/null
-X=($(older defaults "${STAMP}"))
-if [[ "${#X[*]}" != 0 ]]; then
+X="$(older defaults "${STAMP}")"
+if [[ -n "${X}" ]]; then
     echo "Generated defaults files did not change after touching code-generator file:"
     echo "  ${X[*]:-(none)}"
```

Comment:
did you miss this one?

Reply:
Yes, I missed that. Good catch! Strange that shellcheck didn't complain... I'll fix and resubmit.